index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+  </head>
+  <body>
+    <header id="header">
+      <nav id="nav-bar" class="header__nav-bar">
+        <a href="#header" class="nav-link">Hard Rock</a>
+        <a href="#email" class="nav-link">mmmm@Mverse</a>
+        <a href="#submit" class="nav-link">Submit Form</a>
```

Comment:
For semantic reasons => Consider inserting every anchor -element- within a `<li>` and this within either an unordered `<ul>` or ordered `<ol>` list.

e.g.
```html
        <ul>
          <li><a href="" class="nav-link"></a></li>
          <li><a href="" class="nav-link"></a></li>
          <li><a href="" class="nav-link"></a></li>
        </ul>
```
or

```html
        <ol>
          <li><a href="" class="nav-link"></a></li>
          <li><a href="" class="nav-link"></a></li>
          <li><a href="" class="nav-link"></a></li>
        </ol>
```